frontend/src/features/research/activities/ActivityTray/styles.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+
+export const TrayHeader = styled.h3`
+  font-size: 2rem;
+  padding: 0.8rem;
+  border-bottom: 0.4rem solid ${props => props.theme.css.primaryColor};
+  border-top-left-radius: 1rem;
+  border-top-right-radius: 1rem;
+  margin-bottom: 3.2rem;
+  background-color: ${props => props.theme.css.primaryColor};
+  color: white;
+  font-family: '' BCSans ',Fallback,sans-serif';
+`;
+
+//place high z-index on grandparent so that negative z-index tray still draws over map(but under navbar parent).
+export const ZIndexWrapper = styled.div`
+  z-index: 1000;
```

Comment:
would prefer that this is removed unless absolutely necessary.

Reply:
removed